test/tests/mfem/kernels/darcy.i

```diff
@@ -0,0 +1,113 @@
+[Mesh]
+  type = MFEMMesh
+  file = ../mesh/star.mesh
+  uniform_refine = 4
+[]
+
+[Problem]
+  type = MFEMProblem
+[]
+
+[Functions]
+  [exact_velocity]
+    type = ParsedVectorFunction
+    expression_x = '-exp(x) * sin(y)'
+    expression_y = '-exp(x) * cos(y)'
+  []
+  [exact_pressure]
+    type = ParsedFunction
+    expression = 'exp(x) * sin(y)'
+  []
+  [exact_pressure_rhs]
+    type = ParsedFunction
+    expression = '-exp(x) * sin(y)'
+  []
+[]
+
+[FESpaces]
+  [HDivFESpace]
+    type = MFEMVectorFESpace
+    fec_type = RT
+    fec_order = SECOND
+  []
+  [L2FESpace]
+    type = MFEMScalarFESpace
+    fec_type = L2
+    fec_order = SECOND
```

Comment:
this is LBB stable with equal order?

Reply:
MFEM uses a different convention for the order of RT spaces: second, really means the space includes up to third order polynomials.